OpencartTest/src/test/ReviewTest.java

```diff
@@ -45,7 +47,7 @@ public void setUp1() {
     @Parameters({"nameOfAuthor"})
     public void tearDown(ITestResult result, String nameOfAuthor) throws Exception {
         if (result.getStatus() == ITestResult.FAILURE) {
-            Utility.getScreenshot(Driver.getDriver());
+            Screenshot.run(result,driver);
```

Comment:
you guys need to have common teardown method that will try and make a screenshot on failure to avoid code duplication

Reply:
so I need to do this screenshot method without parameters or just use it in TestRunner which is a common class?.